internal/net/client_grpc_dkg.go

```diff
@@ -8,78 +8,57 @@ import (
 	"github.com/drand/drand/protobuf/drand"
 )
 
-func (g *grpcClient) Propose(ctx context.Context, p Peer, in *drand.ProposalTerms, _ ...grpc.CallOption) (*drand.EmptyResponse, error) {
-	var resp *drand.EmptyResponse
+func (g *grpcClient) Command(ctx context.Context, p Peer, in *drand.DKGCommand, _ ...grpc.CallOption) (*drand.EmptyDKGResponse, error) {
```

Comment:
why do you need the `_ ...grpc.CallOption` ?

Can't you change the interface?

Reply:
The construction of the `grpcClient` [here](https://github.com/drand/drand/blob/cc287687d4b038941982508663fb801e31de1fe8/internal/net/client_grpc.go#L58) expects them in a generic way and all other interfaces have them